modules/installation-special-config-encrypt-disk.adoc

```diff
@@ -0,0 +1,155 @@
+Tang encryption mode is only supported for bare metal installs.
+
+Follow one of the two procedures below to enable disk encryption for the nodes in your cluster.
+
+== Enabling TPM v2 disk encryption
+Use this procedure to enable TPM v2 mode disk encryption during {product-title} deployment.
+
+.Procedure
+
+. Check to see if TPM v2 encryption needs to be enabled in the BIOS on each node.
+This is required on most Dell systems.
+
+. Generate the Kubernetes manifests for the cluster:
++
+----
+$ ./openshift-install create manifests --dir=<installation_directory>
+----
+
+. In the `openshift` directory, create a file (for example, 99_openshift-tpmv2-encryption.yaml)
+to define the `/etc/clevis.json` file. Here's what the contents of that file should include:
++
+----
+ "storage": {
+    "files": [
+      {
+        "filesystem": "root",
+        "group": {},
+        "path": "/etc/clevis.json",
+        "user": {},
+        "contents": {
+          "source": "data:text/plain;base64,e30K"
+        },
+        "mode": 420
+      }
+    ]
+  }
+----
+
++
+
+. Make a backup copy of the yaml file. You should do this because the file will be deleted when you create the cluster.
+
+. Obtain the Ignition config files:
++
+----
+$ ./openshift-install create ignition-configs --dir=<installation_directory> <1>
+----
+<1> For `<installation_directory>`, specify the same installation directory.
++
+The following files are generated in the directory:
++
+----
+.
+├── auth
+│   ├── kubeadmin-password
+│   └── kubeconfig
+├── bootstrap.ign
+├── master.ign
+├── metadata.json
+└── worker.ign
+----
+
+== Enabling Tang disk encryption
+Use this procedure to enable Tang mode disk encryption during {product-title} deployment.
+
+.Procedure
+
+. Set up or access an existing Tang server. See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption_security-hardening#network-bound-disk-encryption_configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption[Network-bound disk encryption]
+for instructions. See link:https://youtu.be/2uLKvB8Z5D0[Securing Automated Decryption New Cryptography and Techniques]
+for a presentation on Tang.
+
+. Add kernel arguments to configure networking when you do the RHCOS installations for your cluster.
+If you miss this step, the second boot will fail.
+For example, to configure DHCP networking, identify `ip=dhcp`
+or set static networking when you add parameters to the kernel command line.
+See
+link:https://docs.openshift.com/container-platform/4.3/installing/installing_bare_metal/installing-bare-metal.html#creating-machines-bare-metal[Creating Red Hat Enterprise Linux CoreOS (RHCOS) machines using an ISO image] for details.
+
+. Generate the thumbprint. Install the clevis package (if not already installed) and generate a thumbprint
```

Comment:
package, if it is not already installed, and

Reply:
Done